packages/main/src/SideNavigationItem.hbs

```diff
@@ -0,0 +1,27 @@
+<div
+    class="{{classessni.root}}"
```

Comment:
"classesni" is a bit confusing. It works only because in `litVisitor2.js` our check is not very strict:
```js
const hasCalculatingClasses = path.includes("context.classes");
```
so if it was called `niclasses` for example, it wouldn't match and the object would not be converted to classmap.

I think it's best to make this "classes" to be consistent with the other components.